P0009/P0009.bs

```diff
@@ -1018,56 +1083,63 @@ Table � — Layout mapping policy and layout mapping requirements
 <tr>
   <td>`m(i...)`</td>
   <td>`E::index_type`</td>
-  <td>*Returns:* Mapping of a multi-index `i...` </td>
-  <td>*Requires:* `0 <= m(i...)` </td>
+  <td>*Returns:* Mapping of a multi-index `i...` . </td>
+  <td>*Requires:* 0 <= `m(i...)`. </td>
 </tr>
 <tr>
   <td>`m.required_span_size()`</td>
   <td>`E::index_type`</td>
-  <td>*Returns:* one plus the maximum value of `m(i...)`.</td>
+  <td>*Returns:* If the multidimensional index space that `e` defines is empty, then zero,
+      else 1 plus the maximum value of `m(i...)` for all `i...` in `e`.</td>
   <td></td>
 </tr>
 <tr>
   <td>`m.is_unique()`</td>
   <td>`bool`</td>
-  <td>*Returns:* `true` if `m(i...)!=m(j...)` for every `i...!=j...` </td>
+  <td>*Returns:* `true` if for every `i...!=j...`, `m(i...)!=m(j...)` is `true`. </td>
   <td></td>
 </tr>
 <tr>
   <td>`m.is_contiguous()`</td>
   <td>`bool`</td>
-  <td>*Returns:* `true` if the set of values defined by `m(i)...` is equal to the set of values consisting of `0`...`m.required_span_size()-1` </td>
+  <td>*Returns:* `true` if the set defined by `m(i)...` equals the set {0, ..., `m.required_span_size()-1`}. </td>
   <td></td>
 </tr>
 <tr>
   <td>`m.is_strided()`</td>
   <td>`bool`</td>
-  <td>*Returns:* `true` if `m(j...) - m(i...) = s[r]` when all members of `j...` and `i...` are equal except for exactly one `r`*th* member such that `j[r]==i[r]+1`. `s[r]` is the *stride* of ordinate `r`.</td>
+  <td>*Returns:* `true` if there exists an integer <math>&sigma;<sub>r</sub></math>
+       such that, for all `j...` and `i...` in `e`,
+       if all members of `j...` and `i...` are equal
+       except for exactly one r-th member where
+       <math>j<sub>r</sub></math> equals <math>i<sub>r</sub></math> + 1, 
+       then `m(j...) - m(i...)` equals <math>&sigma;<sub>r</sub></math>.</td>
```

Comment:
Does this also need to say `sigma_r > 1`?

Reply:
`layout_left``mdspan` are strided (`is_strided()` is `true`).  Consider the left-most index.  There, `m(j...) - m(i...)` equals 1.  Thus, <math>sigma<sub>r</sub></math> may be 1 in this case.

Whether <math>sigma<sub>r</sub></math> may be zero relates to the issue of degenerate mdspan, #78.